charts/kafka/charts/zookeeper/templates/statefulset.yaml

```diff
@@ -32,222 +32,198 @@ spec:
       release: {{ .Release.Name }}
       component: server
   updateStrategy:
```

Comment:
Can you update the licence header. Specifically add the crown copyright and change the 'copied from ...' to 'Originally based on ...'